detectron2/evaluation/fast_eval_api.py

```diff
@@ -119,3 +119,10 @@ def accumulate(self):
         self.eval["scores"] = np.array(self.eval["scores"]).reshape(self.eval["counts"])
         toc = time.time()
         logger.info("COCOeval_opt.accumulate() finished in {:0.2f} seconds.".format(toc - tic))
+
+
+try:
+    from detectron2 import _C
+except ImportError:
+    _msg = "detectron2 is not compiled successfully, please build following the instructions!"
+    COCOeval_opt = create_dummy_class("COCOeval_opt", "detectron2._C", _msg)
```

Comment:
Can we fall back to non-accelerated version?

Reply:
Added fallback with official COCO evaluation.